src/app/components/CrossBack/styles.ts

```diff
@@ -1,12 +1,17 @@
-import { extraLightGray, white } from '@constants/colors';
 import { StyleSheet } from 'react-native';
+import { extraLightGray, white } from '@constants/colors';
+import { isIos } from '@constants/platform';
+
+const PADDING_VERTICAL = isIos ? 8 : 4;
+const PADDING_HORIZONTAL = isIos ? 10 : 8;
 
 const styles = StyleSheet.create({
   container: {
     marginLeft: 10,
-    borderRadius: 50,
+    borderRadius: 10,
```

Comment:
for repeated values ​​you can create a variable

Reply:
added COMMON_SPACING=10;